src/components/Dropdown/Dropdown.tsx

```diff
@@ -40,6 +40,7 @@ export interface DropdownProps
   inline?: boolean;
   label: ReactNode;
   theme?: DeepPartial<FlowbiteDropdownTheme>;
+  renderTrigger?: () => ReactNode;
```

Comment:
@paghar thank you.

I think that it is almost there. I think that it is important to pass the `theme` as a function parameter, so it can be used to access the theme styles, as, for example... `theme.arrowIcon`.

So probably, I would replace this line with:

```ts
renderTringger?: (theme: FlowbiteDropdownTheme) => ReactNode;
```

Also:

- [ ] Update the documentation example
- [ ] Include some test cases for it

And then, it is good to merge.